test/publicPath-option.test.js

```diff
@@ -164,87 +162,140 @@ describe('when applied with `publicPath` and `prefixPublicPathWithWebpackPublicP
 
     expect({ assets, source }).toMatchSnapshot();
   });
+});
 
-  // notice that this case will produce invalid urls if __webpack_public_path__ is set to an absolute url
-  it('matches snapshot for `{String}` value as URL', async () => {
-    const config = {
-      loader: {
-        test: /(png|jpg|svg)/,
-        options: {
-          publicPath: 'https://cdn.com/',
-          prefixPublicPathWithWebpackPublicPath: true,
+describe('when applied with `publicPath` and `postTransformPublicPath` option', () => {
```

Comment:
Let's move this tests (all related to `postTransformPublicPath`) to `postTransformPublicPath-option.text.js`